src/libsync/networkjobs.h

```diff
@@ -498,6 +498,24 @@ private slots:
     bool finished() override;
 };
 
+/**
+ * @brief A basic file manipulation job
+ * @ingroup libsync
+ */
+class OWNCLOUDSYNC_EXPORT SimpleFileManipulationNetworkJob : public AbstractNetworkJob
```

Comment:
I would go with `SimpleFileJob` or `CustomVerbFileJob` like we have `DeleteFileJob`
by the way could you refactor existing very similar code like this `DeleteFileJob` class to share common code ?